### ISSUE TITLE
--quiet switch is not honored

### ISSUE THREAD

**Issue by maddenp** [conda/conda-build#2604]
```
% conda build --help | grep quiet
  -q, --quiet           do not display progress bar
% conda --version
conda 4.4.4
% conda-build --version
conda-build 3.2.0
% conda build --channel conda-forge --channel defaults --override-channels --quiet .
[ elided ]
Downloading and Extracting Packages
libgfortran 3.0.0: ######## | 100% 
libpng 1.6.34: ############ | 100% 
openblas 0.2.20: ########## | 100% 
blas 1.1: ################# | 100% 
backports 1.0: ############ | 100% 
configparser 3.5.0: ####### | 100% 
enum34 1.1.6: ############# | 100% 
lazy-object-proxy 1.3.1: ## | 100% 
mccabe 0.6.1: ############# | 100% 
numpy 1.13.3: ############# | 100% 
pycodestyle 2.3.1: ######## | 100% 
wrapt 1.10.11: ############ | 100% 
autopep8 1.3.3: ########### | 100% 
eccodes 2.5.0: ############ | 100% 
isort 4.2.15: ############# | 100% 
python-eccodes 2.5.0: ##### | 100% 
astroid 1.5.3: ############ | 100% 
grib-writing 0.1.2: ####### | 100% 
pylint 1.7.1: ############# | 100% 
nwp-pydev 0.1.0: ########## | 100% 
```
Aren't those the progress bars `--quiet` is supposed to suppress? I'd like not to see them in logfiles for automated deployments. I also ran w/o `--quiet` to see if there was something else that this might refer to, and there seems to be an animated spinner, too, but it is also not suppressed by the `--quiet` switch.

If I do e.g. `conda create -y -n numpy numpy --quiet`, both the progress bars and the spinner are indeed suppressed. Insofar as conda-build is sharing underlying code, could it be that the quiet option is not being passed through correctly?

**Comment by msarahan** [comment]
This is newly broken with conda 4.4.  We'll look into a fix for the next release.

**Comment by Bo887** [comment]
This seems to be fixed with conda 4.6.4 and conda-build 3.17.8.

**Comment by jjhelmus** [comment]
@Bo887 Thanks for checking this.  Closing.
